src/components/core/authentication/MultiStepSignup.tsx

```
@@ -405,29 +454,46 @@ export default function MultiStepSignup({ onBack, onSubmit, isLoading }: MultiSt
       case 1:
         return !!getValues('role') && !errors.role;
       case 2:
-        // Account setup: name, email, password
         const name = getValues('name');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)